### DISCUSSION TITLE
is current wrapper update to latest version ?  according to c zstd

### DISCUSSION THREAD

**Discussion by Tovi163** [klauspost/compress#714] [Q&A] [answered]
hi  all
is current wrapper update to latest version ?  according to c zstd
https://github.com/facebook/zstd/releases 

The zstd from DataDog use the [version](https://github.com/DataDog/zstd#zstd-go-wrapper) "The current headers and C files are from v1.5.0 (Commit [10f0e699](https://github.com/facebook/zstd/releases/tag/v1.5.0))."

what i want to know is? current warpper's version ? 1.5.0 or 1.5.2 

thanks a  lot

**Comment by klauspost** [comment, marked as answer]
This is not a wrapper :)

**Reply by Tovi163** [reply]
> This is not a wrapper :)

I see, thank you
Is it necessary to support more compression levels, in order to achieve finer-grained compression control

eg,  Make the level=5，to get relative high rate of compress but less CPU cost

**Reply by klauspost** [reply]
You have `WithEncoderLevel(SpeedBetterCompression)` for that. It provides better compression than default, aimed at about a 50% speed reduction.

In practice only a few encoding levels are used, so I've focused on optimizing 4 levels.
